Libs/Core/ctkUtils.cpp

```diff
@@ -79,6 +79,26 @@ void ctk::stlVectorToQList(const std::vector<std::string>& vector,
   std::transform(vector.begin(),vector.end(),std::back_inserter(list),&QString::fromStdString);
 }
 
+//------------------------------------------------------------------------------
+QSet<QString> ctk::qListToQSet(const QStringList& list)
+{
+#if (QT_VERSION >= QT_VERSION_CHECK(5, 14, 0))
+  return QSet<QString>(list.begin(), list.end());
+#else
+  return list.toSet();
+#endif
+}
+
+//------------------------------------------------------------------------------
+QStringList ctk::qSetToQList(const QSet<QString>& set)
```

Comment:
Since the input is a QStringList (not any list), the method name should include QStringList. It also make sense to use name similar to what Qt uses. Based on these, qStringListFromSet would be more appropriate.